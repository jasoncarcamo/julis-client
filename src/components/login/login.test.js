import React from "react";
import ReactDOM from "react-dom";
import Login from "./LogIn";
import {BrowserRouter} from "react-router-dom";
import Enzyme from "enzyme";
import renderer from "react-test-renderer";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter()});


describe("<Login/>", ()=>{
    it("Renders login compeonent", ()=>{
        const div = document.createElement("div");

        ReactDOM.render(<BrowserRouter><Login/></BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it("create snapshot for login component", ()=>{
        const tree = renderer.create(<BrowserRouter><Login/></BrowserRouter>).toJSON();

        expect(tree).toMatchSnapshot();
    })

})