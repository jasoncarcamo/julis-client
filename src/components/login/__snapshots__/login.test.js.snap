// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> create snapshot for login component 1`] = `
<section
  id="login_section"
>
  <form
    id="login_form"
    onSubmit={[Function]}
  >
    <fieldset>
      <label
        htmlFor="user_name"
      >
        Mobile Number:
      </label>
      <input
        id="user_name"
        onChange={[Function]}
        type="text"
        value=""
      />
      <label
        htmlFor="password"
      >
        Password:
      </label>
      <input
        id="password"
        onChange={[Function]}
        type="password"
        value=""
      />
      <button
        className="login_button"
        id="login_submit"
        type="submit"
      >
        Log In
      </button>
      
    </fieldset>
  </form>
</section>
`;
